tests/Shared/RepoTesting/Directory.Packages.Helix.props

```diff
@@ -3,86 +3,86 @@
-    <PackageVersion Include="Aspire.Azure.Messaging.EventHubs" Version="9.5.0" />
-    <PackageVersion Include="Aspire.Azure.Messaging.ServiceBus" Version="9.5.0" />
-    <PackageVersion Include="Aspire.Azure.Messaging.WebPubSub" Version="9.5.0" />
-    <PackageVersion Include="Aspire.Azure.Search.Documents" Version="9.5.0" />
-    <PackageVersion Include="Aspire.Azure.Security.KeyVault" Version="9.5.0" />
-    <PackageVersion Include="Aspire.Azure.Storage.Blobs" Version="9.5.0" />
-    <PackageVersion Include="Aspire.Azure.Storage.Queues" Version="9.5.0" />
-    <PackageVersion Include="Aspire.Azure.Npgsql.EntityFrameworkCore.PostgreSQL" Version="9.5.0" />
-    <PackageVersion Include="Aspire.Confluent.Kafka" Version="9.5.0" />
-    <PackageVersion Include="Aspire.Hosting" Version="9.5.0" />
-    <PackageVersion Include="Aspire.Hosting.AppHost" Version="9.5.0" />
-    <PackageVersion Include="Aspire.Hosting.Azure" Version="9.5.0" />
-    <PackageVersion Include="Aspire.Hosting.Azure.AppConfiguration" Version="9.5.0" />
-    <PackageVersion Include="Aspire.Hosting.Azure.AppContainers" Version="9.5.0" />
-    <PackageVersion Include="Aspire.Hosting.Azure.ApplicationInsights" Version="9.5.0" />
-    <PackageVersion Include="Aspire.Hosting.Azure.CognitiveServices" Version="9.5.0" />
-    <PackageVersion Include="Aspire.Hosting.Azure.CosmosDB" Version="9.5.0" />
+    <PackageVersion Include="Aspire.Azure.Data.Tables" Version="9.5.1" />
+    <PackageVersion Include="Aspire.Azure.Messaging.EventHubs" Version="9.5.1" />
+    <PackageVersion Include="Aspire.Azure.Messaging.ServiceBus" Version="9.5.1" />
+    <PackageVersion Include="Aspire.Azure.Messaging.WebPubSub" Version="9.5.1" />
+    <PackageVersion Include="Aspire.Azure.Search.Documents" Version="9.5.1" />
+    <PackageVersion Include="Aspire.Azure.Security.KeyVault" Version="9.5.1" />
+    <PackageVersion Include="Aspire.Azure.Storage.Blobs" Version="9.5.1" />
+    <PackageVersion Include="Aspire.Azure.Storage.Queues" Version="9.5.1" />
+    <PackageVersion Include="Aspire.Azure.Npgsql.EntityFrameworkCore.PostgreSQL" Version="9.5.1" />
+    <PackageVersion Include="Aspire.Confluent.Kafka" Version="9.5.1" />
+    <PackageVersion Include="Aspire.Hosting" Version="9.5.1" />
+    <PackageVersion Include="Aspire.Hosting.AppHost" Version="9.5.1" />
+    <PackageVersion Include="Aspire.Hosting.Azure" Version="9.5.1" />
+    <PackageVersion Include="Aspire.Hosting.Azure.AppConfiguration" Version="9.5.1" />
+    <PackageVersion Include="Aspire.Hosting.Azure.AppContainers" Version="9.5.1" />
+    <PackageVersion Include="Aspire.Hosting.Azure.ApplicationInsights" Version="9.5.1" />
+    <PackageVersion Include="Aspire.Hosting.Azure.CognitiveServices" Version="9.5.1" />
+    <PackageVersion Include="Aspire.Hosting.Azure.CosmosDB" Version="9.5.1" />
     <PackageVersion Include="Aspire.Hosting.Azure.ContainerRegistry" Version="$(PackageVersion)" />
```

Comment:
Does `$(PackageVersion)` not work? also, whichever option we use, all the references should be updated for that.